src/class.h

```diff
@@ -47,10 +57,11 @@ class Thumbnailer {
   };
   std::vector<FrameData> frames;
   WebPAnimEncoder* enc = NULL;
-  WebPData webp_data;
   WebPAnimEncoderOptions anim_config;
   WebPConfig config;
-  int loop_count;
+  int loop_count = 0;
 };
 
 }  // namespace libwebp
+
+#endif  // THUMBNAILER_SRC_CLASS_H_
```

Comment:
Please make sure your file has an extra line at the end

Reply:
I think we do have an extra line but it doesn't show up here.